### PR TITLE
Enable services

### PR DESCRIPTION
**Pull request by jackTheRipper**
** Do **

Update the README (add badges). Add configuration for Travis CI. Add MAINTAINERS file. Add Makefile for publishing

### CODE REVIEW

**Comment by fahad19** [comment]
npm (until v2 at least) warned about not using any `serve` script, and use `start` instead. 

side note: `lint` is more common as script name than `linter`.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: node_js
+
+node_js:
+  - 6
+  - 8
```
Comment (fahad19) [inline .travis.yml]: latest LTS?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+script:
+  - 'npm run linter'
+  - 'npm run test'
+
+notifications:
+  email: true
```
Comment (fahad19) [inline .travis.yml]: `.editorconfig`? :P

File: .travis.yml (hunk 1 above)
Comment (jackTheRipper) [inline .travis.yml]: Already covered by the `6`. If you look at the logs it download 6.11 (latest LTS)

**Comment by jackTheRipper** [comment]
@fahad19 : all done

**Pull request merged**